patches/minecraft/net/minecraft/entity/passive/EntityVillager.java.patch

```diff
@@ -120,16 +120,26 @@
              this.field_175562_bw = 1;
          }
  
-@@ -552,7 +596,7 @@
+@@ -552,16 +596,9 @@
  
          int i = this.field_175563_bv - 1;
          int j = this.field_175562_bw - 1;
 -        EntityVillager.ITradeList[][] aentityvillager$itradelist1 = aentityvillager$itradelist[i];
-+        EntityVillager.ITradeList[][] aentityvillager$itradelist1 = this.getProfessionForge().getCareer(i).getTrades();
```

Comment:
The hook was designed to return a dual array to prevent the patch from being large. So ya, don't do that.